firefox-ios/Client/Frontend/Browser/Toolbars/Models/AddressToolbarContainerModel.swift

```diff
@@ -18,13 +18,15 @@ class AddressToolbarContainerModel {
         let locationViewState = LocationViewState(
             clearButtonA11yId: AccessibilityIdentifiers.Browser.AddressToolbar.clear,
             clearButtonA11yLabel: .AddressToolbar.LocationClearButtonA11yLabel,
+            lockIconButtonA11yId: AccessibilityIdentifiers.Browser.AddressToolbar.lockIcon,
+            lockIconButtonA11yLabel: "",
```

Comment:
Please add an a11y label to our strings or use the existing one (if available). This should not stay empty as it might get forgotten otherwise.